pyproject.toml

```diff
@@ -54,10 +54,9 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.mypy]
 files = ["rich"]
-warn_unused_configs = true
```

Comment:
This is included in strict, which I didn't realize until checking the `mypy --help` output just now.